src/main/webapp/script.js

```diff
@@ -60,6 +60,54 @@ function checkPermissions() {
   });
 }
 
+function hideAccessForm() {
+  this.style.display = 'none';
+  document.getElementById('content').style.display = 'block';
+}
+
+function makeFormDissapear() {
```

Comment:
Really tiny typo but I think this should be makeFormDisappear()

Reply:
Thank you! Solved in https://github.com/googleinterns/step229-2020/pull/70/commits/1d3f009d4f51b391b23a70e256daec9e669af1f4